src/provider/rpc.ts

```diff
@@ -46,7 +45,7 @@ export const getDefaultNodeUrl = (networkName?: NetworkName, mute: boolean = fal
   if (!mute)
     // eslint-disable-next-line no-console
     console.warn('Using default public node url, please provide nodeUrl in provider options!');
-  const nodes = networkName === NetworkName.SN_MAIN ? RPC_MAINNET_NODES : RPC_GOERLI_NODES;
+  const nodes = networkName ? RPC_NODES[networkName] : RPC_NODES[NetworkName.SN_GOERLI];
```

Comment:
```suggestion
  const nodes = RPC_NODES[networkName ?? NetworkName.SN_GOERLI];
```
Could use the shorter expression above or set the default value of the `networkName` parameter to `NetworkName.SN_GOERLI`.